Audio-Transcription-Chrome/popup.js

```diff
@@ -129,7 +119,6 @@ document.addEventListener("DOMContentLoaded", function () {
     startButton.disabled = isCapturing;
     stopButton.disabled = !isCapturing;
     useServerCheckbox.disabled = isCapturing; 
-    useMultilingualCheckbox.disabled = isCapturing;
     modelSizeDropdown.disabled = isCapturing;
     
```

Comment:
```suggestion
taskDropdown.disabled = isCapturing

```

I guess it was missed initially.